### ISSUE TITLE
Add specific versions to install packages in defaults/main.yml

### ISSUE THREAD

**Issue by richardboswell** [vmware/ansible-role-ansible#2]
Wherever possible we need to add specific version of each of the binaries installed. For example, python-pip needs to be at 8.1.2 now to install Ansible 2.1.1.0 as we didn't define the Ansible version.

**Comment by tompscanlan** [comment]
yes... need to add a pin for the version.  also need to allow updating on re-run if not at the pinned version.
